readme/api.md

```diff
@@ -119,6 +119,9 @@ Gets note with ID :id
 
 Gets all the tags attached to this note.
 
+## GET /notes/:id/resources
+
+Gets all the resources attached to this note.
```

Comment:
To be consistent with the rewst f the file, there should be an empty line between the text and the next header.